jax/experimental/jax2tf/examples/tf_js/quickdraw/utils.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from jax import numpy as jnp # type: ignore
+
+import numpy as np # type: ignore
+import os
+import requests # type: ignore
+
+def download_dataset(dir_path, nb_classes):
+
+  if not os.path.exists(dir_path):
+    os.mkdir(dir_path)
```

Comment:
I think that `os.makedirs` will create the directory and its parents, if needed.